boot/rootlesskit.sh

```diff
@@ -54,6 +59,19 @@ else
 		/etc/containerd /etc/containers /etc/crio \
 		/etc/kubernetes
 
+	if [[ "$U7S_CGROUP_ENABLED" = "1" ]]; then
+		# we are in cgroupns, we need to mount our own cgroupfs
+		mount -t tmpfs none /sys/fs/cgroup
+		mount -t cgroup2 none /sys/fs/cgroup
+		cd /sys/fs/cgroup
+		(
+			# evacuate existing procs so that we can enable all controllers including threaded ones
```

Comment:
@giuseppe 
Q: is it valid to move the `rootlesskit` process running in `/user.slice/user-1001.slice/user@1001.service/u7s-rootlesskit.service` to `/user.slice/user-1001.slice/user@1001.service/u7s-rootlesskit.service/evacuation`? 

The rootlesskit service is defined like this:
```
[Unit]
Description=Usernetes RootlessKit service
PartOf=u7s.target

[Service]
ExecStart=/home/suda/gopath/src/github.com/rootless-containers/usernetes/boot/rootlesskit.sh
Delegate=cpu cpuset io memory pids
WorkingDirectory=/home/suda/gopath/src/github.com/rootless-containers/usernetes
EnvironmentFile=/home/suda/.config/usernetes/env
Restart=on-failure
LimitNOFILE=65536
```